languages.yaml

```diff
@@ -2331,6 +2331,34 @@ Vala:
       - magenta
       - white
     chip: '#A56DE2'
+Verilog:
+  type: programming
+  ascii: |
+    {0}           _.._   _.._   _.._
+    {0}           _.._   _.._   _.._
+    {0}           _.._   _.._   _.._
+    {0}       _......................_
+    {0}      _.{1} -----          ----- {0}._
+    {0}  _..._.{1}  ---            ---  {0}._..._
+    {0}  _..._.{1}   ---          ---   {0}._..._
+    {0}      _.{1}    ---        ---    {0}._
+    {0}      _.{1}     ---      ---     {0}._
+    {0}  _..._.{1}      ---    ---      {0}._..._
+    {0}  _..._.{1}      ---    ---      {0}._..._
+    {0}      _.{1}       ---  ---       {0}._
+    {0}      _.{1}        --  --        {0}._
+    {0}  _..._.{1}        -----         {0}._..._
+    {0}  _..._.{1}         ---          {0}._..._
+    {0}      _.{1}          -          {0}.._
+    {0}       _......................_
+    {0}         _.._   _.._   _.._
+    {0}         _.._   _.._   _.._
+    {0}         _.._   _.._   _.._
+  colors:
+    ansi:
+      - white
+      - magenta
+    chip: '#C5C2FA'
```

Comment:
Should be [#b2b7f8](https://github.com/github/linguist/blob/4ea1b93800c27ecb2221d4a9b19d8881e9197544/lib/linguist/languages.yml#L6875)